pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't commit this file, it's generated by gwt-api-generator

Reply:
i added the changes to the gwt-api-generator to generate the pom with the required changes

Reply:
pom.xml now is removed